src/renderer/containers/App/test/App.test.ts

```diff
@@ -64,4 +64,35 @@ describe("App", () => {
             });
         });
     });
+    it("dispatches updateUploadProgressInfo when multifile progress is updated", () => {
+        // Arrange
+        const fssJob: JSSJob = {
+            created: new Date(),
+            jobId: "foo123",
+            jobName: "test_file.txt",
+            modified: new Date(),
+            originationHost: "dev-aics-fup-001",
+            service: Service.FILE_STORAGE_SERVICE,
+            updateParent: false,
+            user: "fakeuser",
+            status: JSSJobStatus.WORKING,
+            serviceFields: {
+                fileSize: 100,
+                subfiles: {
+                    'fileid1': 10,
+                    'fileid2': 10,
+                    'fileid3': 15
+                },
+            },
+        };
+        let actionPersisted = undefined;
+        const dispatch = (action: Action)=>{
+            actionPersisted = action;
+        };
+        const expectedAction = updateUploadProgressInfo(fssJob.jobId, { bytesUploaded: 35, totalBytes: fssJob.serviceFields?.fileSize, step: Step.THREE })
```

Comment:
Is step 3 the 'still uploading' step?

Reply:
Step 3 is the "post-upload MD5" step, but I don't believe any of the steps are labeled in a way that the users could tell what's going on.